src/client/java/teammates/client/scripts/RepairFeedbackSessionNameWithExtraWhiteSpace.java

```diff
@@ -0,0 +1,147 @@
+            if (isPreview) {
+                System.out.println("There are/is " + numberOfFeedbackSessionWithExtraWhiteSpacesInName
+                                   + "/" + feedbackSessionList.size() + " feedback session(s) with extra spaces in name!");
+            } else {
+                System.out.println("" + numberOfFeedbackSessionWithExtraWhiteSpacesInName
+                                   + "/" + feedbackSessionList.size() + " feedback session(s) have been fixed!");
+                System.out.println("Extra space removing done!");
+            }
+        } catch (InvalidParametersException | EntityDoesNotExistException | EntityAlreadyExistsException e) {
+            e.printStackTrace();
+        }
+    }
+    
+    /**
+     * Displays the feedback session.
+     */
+    private void showFeedbackSession(FeedbackSession session) {
+        System.out.println("Feedback Session Name: \"" + session.getFeedbackSessionName() + "\" in " + session.getCourseId());
+    }
+
+    /**
+     * Remove extra spaces in feedback session's name
+     */
+    private void fixFeedbackSession(FeedbackSession session) throws InvalidParametersException, EntityDoesNotExistException, EntityAlreadyExistsException {
+        fixFeedbackQuestionsOfFeedbackSession(session);
+        fixFeedbackResponsesOfFeedbackSession(session);
+        fixFeedbackResponseCommentsOfFeedbackSession(session);
+        
+        FeedbackSessionAttributes sessionAttribute = new FeedbackSessionAttributes(session);
+        feedbackSessionsDb.deleteEntity(sessionAttribute);
+        sessionAttribute.feedbackSessionName = StringHelper.removeExtraSpace(sessionAttribute.getFeedbackSessionName());
+        feedbackSessionsDb.createEntity(sessionAttribute);
+    }
+
+    /**
+     * Fixes feedbackSessionName in FeedbackResponseComments
+     */
+    private void fixFeedbackResponseCommentsOfFeedbackSession(FeedbackSession session) throws InvalidParametersException, EntityDoesNotExistException {
+        List<FeedbackResponseCommentAttributes> feedbackResponseCommentList = feedbackResponseCommentsDb.getFeedbackResponseCommentsForSession(session.getCourseId(),
+                                                                                                                                               session.getFeedbackSessionName());
+        for (FeedbackResponseCommentAttributes comment : feedbackResponseCommentList) {
+            comment.feedbackSessionName = StringHelper.removeExtraSpace(comment.feedbackSessionName);
+            feedbackResponseCommentsDb.updateFeedbackResponseComment(comment);
+        }
+    }
+
+    /**
+     * Fixes feedbackSessionName in FeedbackResponses
+     */
+    private void fixFeedbackResponsesOfFeedbackSession(FeedbackSession session) throws InvalidParametersException, EntityDoesNotExistException {
+        List<FeedbackResponseAttributes> feedbackResponseList = feedbackResponsesDb.getFeedbackResponsesForSession(session.getFeedbackSessionName(), 
```

Comment:
Just like how the error occurred in the issue, won't this fail to get the responses of the session?